assets/js/components/Results.vue

```diff
@@ -1,12 +1,9 @@
 <template>
-  <div class="sidebar-tab-pane results"
-       v-bind:class="{
-        'is-selected': selected,
-        'is-loading': loading
-       }">
-    <ul class="sidebar-tab-result-list" v-if="hasData">
+  <section class="sidebar-section-wrapper results"
+    role="tabpanel">
```

Comment:
The same, class and role.

Actually, why is this section even needed? Isn't the `ul` container enough?